src/components/send/clickToSendComponent.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styles from './send.css';
+import Send from '../send';
+import { fromRawLsk } from '../../utils/lsk';
+
+
+const ClickToSendComponent = props => (
```

Comment:
If you please rename the Send component to SendForm, then you can use Send for this component. TBH, I don't like the name